src/main/java/com/artipie/vertx/VertxSliceServer.java

```diff
@@ -165,6 +165,9 @@ public void close() {
      * @return Completion of request serving.
      */
     private CompletionStage<Void> serve(final HttpServerRequest req) {
+        if (expectContinue(req)) {
+            req.response().writeContinue();
```

Comment:
@g4s8, you need to return from the function since the response is already produced. And a unit test would be welcome. Not sure if that would work as it is now.